vmdb/lib/miq_automation_engine/service_models/miq_ae_service_vm_or_template.rb

```diff
@@ -1,5 +1,8 @@
 module MiqAeMethodService
   class MiqAeServiceVmOrTemplate < MiqAeServiceModelBase
+    require_relative "mixins/miq_ae_service_ems_operations_mixin"
```

Comment:
requiring a mixin in the class definition looks odd to me.
Is there a reason to add it up here?